src/common/DEFENSIVE_BUFFS.js

```diff
@@ -52,6 +56,22 @@ const DEFENSIVE_BUFFS = [
   {
     spell: SPELLS.GHOST_IN_THE_MIST_BUFF,
   },
+  //Brewmaster Monk
+  {
+    spell: SPELLS.IRONSKIN_BREW_BUFF,
```

Comment:
You already added the buffID to abilities so shouldn't be needed here?

Reply:
Ironskinbrew shares a cooldown with Purifyingbrew.
While ISB has already a buffId, it doesn't show up (I guess because the ability is an array).
Adding it to the `DEFENSIVE_BUFFS` was a quick fix.

Reply:
Interesting, I thought that would've just worked with the way array are implemented.

Reply:
It probably only picks the first spell when looking for what to put on cd

Reply:
It doesn't show up because ISB has to either have the `isDefensive`-prop or a defensive category.
`buffSpellId`s that dont match those criterias wont show up as defensive buffs.
Setting ISB as defensive would cause an error, I assume because `spellUsable.isAvailable()` doesn't like arrays as parameter.
I could add an exeption for that in the tracker, but not quite sure if its actually usefull to show ISB as "cooldown".

Reply:
good point, we do expect 100% uptime on it. Only thing would be that it could indicate if the monk was purifying too much if it was not ready. Anyway it sounds like there might be a bug that needs fixing anyway, it should definitely be possible to call `spellUsable.isAvailable()` for spells that use array aswell.